commands/moderation/say.js

```diff
@@ -7,14 +7,18 @@ module.exports = {
     usage: "<input>",
     run: (client, message, args) => {
         message.delete();
-        message.channel.send(args.join(" "));
+        if(args[0])
+            message.channel.send(args.join(" "));
+        message.channel.send("Cannot send an empty message.");
```

Comment:
Lets say our if statement passes as true and it sends the parsed message. What stops the program from running this line? There's no return in the if statement so both messages will be sent correct?